CONTRIBUTING.md

```diff
@@ -0,0 +1,139 @@
+
+## Preparations
+
+After you’ve forked the Dataverse Frontend repository, you should install all npm dependencies.
+
+```bash
+npm install
+```
+
+## Coding Standards
+
+We use [TODO: EasyCodingStandard](https://github.com/symplify/easy-coding-standard) for Dataverse’s code and code examples in the documentation, which follows the [TODO: X: Extended Coding Styles](#).
+
+We use ESLint to automatically check and apply the coding standards to our codebase, reducing the manual work to a minimum.
+
+### Check and apply coding standards
+
+To run all checks, you can run the `lint` script.
+
+```bash
+npm run lint
+```
+
+You can also apply coding style fixes automatically.
+
+```bash
+npm run lint:fix
+```
+
+### Check and apply formatting standards
+
+Launches the prettier formatter. We recommend you to configure your IDE to run prettier on save.
+
+```bash
+npm run format
+```
+
+### Enforcing coding standards using pre-commit hooks
+
+We use [TODO: XX](#) to add pre-commit hooks which automatically check the committed code changes for any coding standard violations.
+
+### Tests
+
+`npm run test:unit` Launches the test runner for the unit tests in the interactive watch mode.
+If you prefer to see the tests executing in cypress you can run `npm run cy:open-unit`
+You can check the coverage with `npm run test:coverage`
+
+`npm run test:e2e` Launches the Cypress test runner for the end-to-end tests.
+If you prefer to see the tests executing in cypress you can run `npm run cy:open-e2e`
+
+### Other commands
+
+There are more commands that we use to ensure code quality. Usually, you won’t need them.
+
+<!-- TODO: -->
+
+## Process
+
+All PRs receive a review from at least one maintainer. We’ll do our best to do that review as soon as possible, but we’d rather go slow and get it right than merge in code with issues that just lead to trouble.
+
+### GitHub reviews & assignments
+
+You might see us assign multiple reviewers, in this case these are OR checks (i.e. either Person1 or Person2) unless we explicitly say it’s an AND type thing (i.e. can both Person3 and Person4 check this out?).
+
+We use the assignee to show who’s responsible at that moment. We’ll assign back to the submitter if we need additional info/code/response, or it might be assigned to a branch maintainer if it needs more thought/revision (perhaps it’s directly related to an issue that's actively being worked on).
+
+Once approved, the lead maintainer for the branch should merge the PR into the `develop` branch.
+
+### Codeowners
+
+We use a [CODEOWNERS](https://github.com/iqss/dataverse-frontend/blob/master/.github/CODEOWNERS) file to define who gets automatic review invitations.
+
+## Ownership
+
+By contributing code you are granting _(`Dataverse`)_ shared ownership of your work. You still own it but _(`Dataverse`)_ will have the right to relicense your work based on our needs & treat this work as if it was developed by a _(`Dataverse`)_ engineer.
```

Comment:
Is this legalese correct?

Reply:
No idea, I just removed that part and used the Dataverse main repository's CONTRIBUTING file as a reference.